src/realm/group.hpp

```diff
@@ -529,6 +529,11 @@ class Group : private Table::Parent {
     /// If this group accessor is the detached state, this function returns
     /// zero.
     size_t compute_aggregated_byte_size(SizeAggregateControl ctrl = SizeAggregateControl::size_of_all) const noexcept;
+    // Return the size taken up by the current commit. This is in contrast to
+    // the number returned by SharedGroup::get_stats() which will return the
+    // size of the last commit done in that SharedGroup. If the commits are the
+    // same, the number will of course be the same.
+    size_t get_used_space() const noexcept;
```

Comment:
I would have spoken in terms of "snapshots" here, but not a show-stopper for me.

Reply:
Why not make this comment Doxygen compliant, as the other ones?